Appraisals

```diff
@@ -8,6 +8,7 @@
 
 appraise 'rails-5.0' do
   gem 'rails', '>= 5.0.0', '< 5.1'
+  gem 'sqlite3', '< 1.4'
```

Comment:
Rails 5.0 breaks on sqlite3 1.4 (recently released)

Reply:
This is similar to #369 but restricts the version pinning to the Rails 5.0 build only.